helm/reana/values.yaml

```diff
@@ -135,6 +135,7 @@ notifications:
 # Accessing the cluster from outside world
 ingress:
   enabled: true
+  host: ""
```

Comment:
Hmm, do we need to add a new field `ingress.host` for this functionality? I'm wondering because we already have `reana_hostname` field defined elsewhere, so if people set it up, we could perhaps reuse that directly.   (And if it is not set up, then use the empty string as default value.)

(Another possibility would be to hardcode `""` as default value inside `helm/reana/templates/reana-workflow-controller.yaml`, without even playing with `reana_hostname`. )

In this way we don't need to introduce the new `ingress.host` field.... Thinking of making changes to 0.9.3 as minimal as possible.